rwandaemr-installer/src/main/resources/scripts/create-service-user.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+## load variables from the vars.txt file
+source vars.txt
+
+home_dir=${parent_dir}
+
+if [ "$env_type" == "demo" -o "$env_type" == "prod" -o "$env_type" == "test" ]; then
```

Comment:
I notice this exact same if statement in 5 places.  We might want to modify this or add additional environment types in the future.  Can we extract this out to something that is defined in a single place and used - either a function call that can be reused or a variable that is set once and passed around?  Something like "isDevEnvironment" is what you are basically looking for (or "isNotDevEnvironment" I guess), right?

It might make sense to set this as an additional variable within your vars.txt that you can check.  Eg. within vars.txt, add an additional line like:

`non_dev_server="$env_type" == "demo" -o "$env_type" == "prod" -o "$env_type" == "test"`

Reply:
I changes this a little, In 2 files, we still have 
`if [ "$env_type" == "demo" -o "$env_type" == "prod" -o "$env_type" == "test" ]; then
  TARGET_DIR="/opt/${parent_dir}/distribution"
else
  TARGET_DIR="${parent_dir}/distribution"
fi`